rpm/Dockerfile

```diff
@@ -7,5 +7,6 @@ RUN rpmdev-setuptree
 
 USER root
 ADD entry.sh /root/
+COPY ./ /root/rpmbuild/SPECS
```

Comment:
Hmm any way we can avoid this? Not crazy about having to build a new container for every build, I was thinking of it more as a reusable build container rather than one containing the actual artifacts to be built. (just an issue of container bloat and build time, etc)